.gitignore

```diff
@@ -7,3 +7,6 @@ export_presets.cfg
 # Mono-specific ignores
 .mono/
 data_*/
+
+# GoDAW specific ignores
+song.gd
```

Comment:
Is this .gitignore needed?

Reply:
As long as i temp save to song.gd .......